_docs/_latest/develop/branch-methodology.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Branch Methodology
+order: 003
+---
+
+#  {{ page.title }}
+
+This page describes the branches in the Forseti repo and how they're used
+and managed.
+
+## Branches
+
+### General Availability (GA) Branches
+
+GA branches are the main branches you'll use to inspect the currently stable &
+production-ready codebase, create a Pull Request (PR), or deploy to production.
+
+* `dev`: development branch
+* `master`: latest stable release; suitable for deploy to production
+* `forsetisecurity.org`: documentations for Forseti website.
+
+The `dev` branch is the starting point where you can create a new PR,
+and where the PR will be merged into the codebase after code review. Although
+there are unit tests, the development branch is still considered to be unstable
```

Comment:
... the development branch is still considered to be unreliable because things can change before they're merged into the `master` branch...

Reply:
Done.